frame/lending/src/lib.rs

```diff
@@ -1311,7 +1317,8 @@ pub mod pallet {
 					total_repay_amount
 				} else {
 					let repay_borrow_amount = total_repay_amount - burn_amount;
-					remaining_borrow_amount -= repay_borrow_amount;
+					remaining_borrow_amount -=
```

Comment:
Isn't this supposed to be `=` instead of `-=` after the modification?

Reply:
yes. so there is bug. right now the pass is not tested, but at least it should not crash